src/pages/nft/NftInformation.tsx

```diff
@@ -0,0 +1,253 @@
+                  <span className="title">DESCRIPTION</span>
+                  <span className="content">{nftState.value?.description}</span>
+                </div>
+                <div className="info-grid">
+                  <div className="content-block">
+                    <span className="title">CREATOR</span>
+                    <span className="grid-content">
+                      {nftState.value?.creatorName ||
+                        truncateText(true, nftState.value?.creatorAddress)}
+                    </span>
+                  </div>
+                  <div className="content-block">
+                    <span className="title">COLLECTION</span>
+                    <div className="grid-content">
+                      {/*
+                        TODO: uncomment the code below when Ghostmarket fix their API, currently is not returning an image for the collection
+                        <img
+                          className="collection-logo"
+                          src={nftState.value?.collection.image}
+                          alt="token-logo"
+                        />
+                      */}
+                      <span>{nftState.value?.collection.name}</span>
+                    </div>
+                  </div>
+                </div>
+                <div className="info-grid">
+                  <div className="content-block">
+                    <span className="title">SYMBOL</span>
+                    <div className="grid-content">{nftState.value?.symbol}</div>
+                  </div>
+                  <div className="content-block">
+                    <span className="title">BLOCKCHAIN</span>
+                    <div className="grid-content">
+                      {returnChainNameAndLogo()}
+                    </div>
+                  </div>
+                </div>
+                <div className="content-block">
+                  <span className="title">CONTRACT</span>
+                  <span className="inline-content">
+                    {nftState.value?.contract}
+                  </span>
+                </div>
+                <div className="content-block">
+                  <span className="title">TOKEN URI</span>
+                  <div className="inline-content">
+                    {nftState.value?.apiUrl || buildGhostMarketUrl()}
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        )}
+        {nftState.isLoading && (
+          <SkeletonTheme
+            color="#21383d"
+            highlightColor="rgb(125 159 177 / 25%)"
+          >
+            <Skeleton height={650} style={{ margin: '50px 0' }} />
+          </SkeletonTheme>
+        )}
+
+        <div className="nft-attributes-container">
+          <p className="section-title">ATTRIBUTES</p>
+          {!nftState.isLoading && (
+            <div className="attributes-grid">
+              {nftState.value && nftState.value?.attributes.length > 0
```

Comment:
talvez seria bom criar uma const para essa validação para facilitar a manutenção

Reply:
se `nftState.value?.attributes` for null, vai dar erro ao pegar o `.length`